src/pages/CreatorPage.tsx

```diff
@@ -135,219 +140,241 @@ const CreatorTabs: React.VFC<{
-                  {member.description && !BraftEditor.createEditorState(member.description).isEmpty() ? (
-                    <BraftContent>{member.description}</BraftContent>
-                  ) : (
-                    <StyledDescription className="ml-3">
-                      {formatMessage(commonMessages.content.noIntroduction)}
-                    </StyledDescription>
-                  )}
+          <div className="col-lg-4 col-12">
+            <OverviewBlock
+              programs={programs}
+              previousPage={`creators_${creatorId}`}
+              podcastPrograms={podcastPrograms}
+              onChangeTab={key => setActiveKey(key)}
+              onSubscribe={() =>
+                isAuthenticated ? onCheckoutModalOpen?.() : setAuthModalVisible && setAuthModalVisible(true)
+              }
+            />
+          </div>
+        </div>
+      ),
+    },
+    {
+      key: 'programs',
+      name: formatMessage(usersMessages.tab.addPrograms),
+      isVisible: programs.length > 0,
+      content: (
+        <>
+          {programs.length === 0 ? (
+            <div className="row">
+              <StyledDescription className="ml-3">{formatMessage(commonMessages.content.noProgram)}</StyledDescription>
+            </div>
+          ) : (
+            <ProgramCollection programs={programs} />
+          )}
+        </>
+      ),
+    },
+    {
+      key: 'activities',
+      name: formatMessage(usersMessages.tab.addActivities),
+      isVisible: activities.length > 0,
+      content: (
+        <div className="row">
+          {activities.length === 0 ? (
+            <StyledDescription className="ml-3">{formatMessage(commonMessages.content.noActivity)}</StyledDescription>
+          ) : (
+            activities
+              .filter(activity => activity.endedAt && activity.endedAt.getTime() > Date.now())
+              .map(activity => (
+                <div key={activity.id} className="col-12 col-lg-4 mb-4">
+                  <ActivityBlock
+                    id={activity.id}
+                    title={activity.title}
+                    coverUrl={activity.coverUrl || undefined}
+                    isParticipantsVisible={activity.isParticipantsVisible}
+                    participantCount={activity.participantCount}
+                    totalSeats={activity.totalSeats}
+                    startedAt={activity.startedAt || undefined}
+                    endedAt={activity.endedAt || undefined}
+                  />
                 </div>
-                <div className="col-lg-4 col-12">
-                  <OverviewBlock
-                    programs={programs}
-                    previousPage={`creators_${creatorId}`}
-                    podcastPrograms={podcastPrograms}
-                    onChangeTab={key => setActiveKey(key)}
-                    onSubscribe={() =>
+              ))
+          )}
+        </div>
+      ),
+    },
+    {
+      key: 'posts',
```

Comment:
<img width="745" alt="image" src="https://user-images.githubusercontent.com/28040359/209619053-f31da187-a98d-436d-bd67-94f4019b5b9e.png">

PostItemCollection use in BlogPage and CreatorPage
https://trello.com/c/LV5YLZPF this feature will influence

Reply:
fixed the blog setting